Week-4/Async_JS/Exchangerates/script.js

```diff
@@ -0,0 +1,16 @@
+async function getExchangeRate(currencyCode) {
+  try {
+    const response = await fetch(`https://api.exchangerate.host/latest`);
+    const output = await response.json();
+    const rates = output.rates;
+    if (rates.hasOwnProperty(currencyCode)) {
+      console.log(rates[currencyCode].toPrecision(5));
+      return rates[currencyCode].toPrecision(5);
+    } else {
+      return null;
+    }
+    //return output.rates.currencyCode;
+  } catch (e) {
+    return 'Error while fetching';
```

Comment:
can return the error or null here
You can override the error message to the string here

Reply:
Fine will do that